packages/espressocash_app/lib/features/conversion_rates/data/jupiter_client.dart

```diff
@@ -3,6 +3,7 @@ import 'package:fast_immutable_collections/fast_immutable_collections.dart';
 import 'package:freezed_annotation/freezed_annotation.dart';
 import 'package:injectable/injectable.dart';
 import 'package:retrofit/dio.dart';
+import 'package:retrofit/error_logger.dart';
```

Comment:
Let's rather replace these 2 lines with:

```dart
import 'package:retrofit/retrofit.dart';
```

Reply:
Sure. The `http` entrypoint was merged too.